src/builder/index.js

```diff
@@ -14,18 +13,45 @@ export default function(options) {
     config,
   } = options
 
+  // Prepare staticAssets path and route
+  let { staticAssets } = options
+  if (staticAssets !== undefined) {
+    if (typeof staticAssets === "string") {
+      staticAssets = {
+        "path": path.join(config.cwd, config.source, staticAssets),
+        "route": staticAssets,
+      }
+    } else if (
+      staticAssets.hasOwnProperty("path") &&
+      staticAssets.hasOwnProperty("route")
+    ) {
+      staticAssets = {
+        "path": path.join(config.cwd, config.source, staticAssets.path),
+        "route": staticAssets.route,
+      }
+    } else {
+      throw new TypeError (
+        `
```

Comment:
Maybe too much whitespace. Indentation will be weird in terminal output don't you think?